src/App.js

```diff
@@ -0,0 +1,107 @@
+import React, { useEffect, useState } from "react";
+import Sidebar from "./components/Sidebar";
+import Editor from "./components/Editor";
+
+import Split from "react-split";
+import { nanoid } from "nanoid";
+
+export const data = [
+  {
+    id: 1,
+    body: `### This just in!
+
+Markdown is an *awesome* way to write **very basic HTML** by typing manually
+
+It can be used for:
+
+1. README files
+1. Slack and Discord messages
+1. **So much more!**
+
+---
+
+Here's a division 👆`,
+  },
+  { id: 2, body: `a` },
+  { id: 3, body: `b` },
+  { id: 4, body: `c` },
+  { id: 5, body: `d` },
+  { id: 6, body: `e` },
+];
+
+export default function App() {
+  const [notes, setNotes] = useState(
+    () => JSON.parse(localStorage.getItem("notes")) || []
+  );
+
+  const [currentNoteId, setCurrentNoteId] = useState(
+    (notes[0] && notes[0].id) || ""
+  );
```

Comment:
Can you format this line, make it more readable, you can use a ternary operator.